webapp/channels/src/packages/mattermost-redux/src/reducers/entities/admin.ts

```diff
@@ -32,6 +32,19 @@ function logs(state: string[] = [], action: AnyAction) {
     }
 }
 
+function allPlainLogs(state: string[] = [], action: AnyAction) {
```

Comment:
Could you remove the logic for `allPlainLogs` and `getAllPlainLogs` now that that's no longer needed?